src/components/Playground.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import AceEditor from 'react-ace'
+
+import 'brace/mode/javascript'
+import 'brace/theme/github'
+
+/**
+ * A prop that is passed to the Playground
+ * @property editorValue - The string content of the react-ace editor
+ * @property updateCode  - A callback function for the react-ace editor's `onChange`
+ */
+export interface IPlaygroundProps {
+  editorValue: string
+  updateCode: (newCode: string) => void
```

Comment:
Usually the convention for event handler begins with `handle<event-name>`, i.e `handleEditorChange`